src/controllers/rest/impl/FileUploadController.ts

```diff
@@ -43,37 +44,9 @@ export class FileUploadController extends BaseRestController {
     public async addEntry(
         @Req() req: Request,
         @Res() res: Response,
-        @QueryParams("expires")
-        @Examples({
-            empty: {
-                summary: "empty",
-                description: "expires according to retention policy",
-                value: "",
-            },
-            "1d": {
-                summary: "1d",
-                description: "expires in 1day",
-                value: "1d",
-            },
-        })
```

Comment:
I would like to keep this. i opaned a question on tsed for it https://github.com/orgs/tsedio/discussions/2631